scripts/copyright_line_check.sh

```diff
@@ -43,5 +43,8 @@ ensure_file_has_copyright_line './setup.py'
 
 # Check every python file in the package's source directory.
 for filename in ./graphql_compiler/**/*.py; do
-    ensure_file_has_copyright_line "$filename"
+    # Don't run copyright check for snapshot files
+    if [[ $filename != */snapshots/snap_*.py ]]; then
```

Comment:
Try adding double quotes around the values you are comparing? If the filename has spaces/special chars, this will cause unexpected behavior.